core/templates/dev/head/pages/maintenance/maintenance.html

```diff
@@ -0,0 +1,105 @@
+<!DOCTYPE html>
+<html ng-app="oppia" lang="<[currentLang]>" ng-controller="Maintenance" itemscope itemtype="http://schema.org/Organization">
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0, user-scalable=yes">
+    <title itemprop="name" translate="Down for Maintenance - Oppia"></title>
+    {% include 'pages/header_css_libs.html' %}
+
+    <script>
+      var GLOBALS = {
```

Comment:
In short, I think it might be best if we remove these entirely from this page (so we can assume the worst).